backend/src/sharedCopy/constants.ts

```diff
@@ -17,9 +17,11 @@ export const CALLBACK_URL =
   process.env.REACT_APP_CALLBACK_URL || process.env.CALLBACK_URL || process.env.NODE_ENV === 'development'
     ? 'https://dev-auth.internal.remote.it/v1/callback'
     : 'https://auth.api.remote.it/v1/callback'
+export const WEBSOCKET_URL = 'wss://ws.remote.it/beta'
```

Comment:
don't need any environtmen variable?

Reply:
Not yet anyway - we can add one if needed - we'll probably also need a production url as well at some point.